examples/sparse/sign.py

```diff
@@ -0,0 +1,82 @@
+import torch
```

Comment:
https://quip-amazon.com/OWYyAbVeGhBz/DGL-Examples

For the path, let's use core/sparse/sign(_pytorch?).py

Reply:
Let's discuss it in the meeting next Tuesday.